### DISCUSSION TITLE
How to automate Snapshot on Frigate to a specific folder

### DISCUSSION THREAD

**Discussion by brunobpsrpg** [blakeblackshear/frigate#9108] [Ask A Question]
I have the Frigate working on a HA. And i was wondering how i can make a timelapse of a day, shooting a snapshot every 5minutes, etc.
There is a simple way of doing it? 

I was wondering putting all the snapshots of that ""month"" as a example, in a specific folder, so i can easily make the timelapse.

Srry for the newbie question, but i am really newbie on this Linux stuff

**Comment by NickM-27** [comment]
Frigate 0.13 supports creating timelapses as an export and this can be automated via the integration 5.0 beta.

You can also output snapshots in HA using the camera service and then create a timelapse separately

**Reply by brunobpsrpg** [reply]
Nice, how can i do this? xD 
There is a tutorial, or something that can explain how to do it?

**Reply by geerlingguy** [reply]
I've also been searching around—I see a lot of references to exporting timelapses, but no mention in the docs and no explicit guides here. I'm not currently using Home Assistant, just running Frigate standalone, and I don't see any options in the UI for it. Is it an API-only thing?

**Reply by NickM-27** [reply]
@geerlingguy time lapse export was added in 0.13 which is currently in release candidate. You'll need to check the pre release docs for it. 

It is available in the UI and via API.

**Reply by geerlingguy** [reply]
@NickM-27 - Thanks, I had just realized that reading through some code... I was like "how is this not in my install!?" then I finally realized I'm still on 0.12 :)

Going to update to 0.13-rc1 which looks like it came out yesterday!

**Reply by NickM-27** [reply]
Yep, lots in 0.13 so be sure to check out the release notes. Happy to answer any questions

**Reply by geerlingguy** [reply]
No more questions right now, but just want to say thanks for the help (and to all the contributors to this wonderful project!). I'm trying to get a couple local instances running finally, and time is my only enemy... trying to see whether the Pi 5 is a reasonable option compared to tiny PCs.

**Comment by brunobpsrpg** [comment]
@NickM-27 i found how to update my Frigate and install the 0.13, but the recording session only have the option to make a timelapse from a calendar date, my idea here is automate and create a timeline with like, 1s every 5minutes, there is a documentation of how can i automate this timelapse?

**Reply by NickM-27** [reply]
You can configure the Timelapse via ffmpeg args to do that if you wanted to, support generally isn't offered for that. To be clear automation is offered via ha beta 5 for timelapses

**Reply by glennlawyer** [reply]
> To be clear automation is offered via ha beta 5 for timelapses

I got this working by 
adding a new automation
"when" -> time and location -> time pattern 
   enter "/5" in the minutes box
"then do" -> Camera -> Take Snapshot
   Targets "choose device"

filename: "/config/www/cam_captures/kitchen_{{ now ().year }}_{{ now ().month }}_{{ now ().day }}_{{ now ().hour }}_{{ now ().minute }}.jpg"
